internal/generate/olm-catalog/csv.go

```diff
@@ -470,13 +471,15 @@ func (g BundleGenerator) updateCSVFromManifests(csv *olmapiv1alpha1.ClusterServi
 	}
 
 	// Update descriptions from the APIs dir.
-	// FEAT(estroz): customresourcedefinition should not be updated for
-	// Ansible and Helm CSV's until annotated updates are implemented.
-	if projutil.IsOperatorGo() {
-		err = updateDescriptions(csv, g.ApisDir)
-		if err != nil {
-			return fmt.Errorf("error updating CSV customresourcedefinitions: %w", err)
-		}
+	// KB_INTEGRATION_TODO(estroz): remove this condition once kubebuilder's CLI is
```

Comment:
```suggestion
	// todo(estroz): [KB_INTEGRATION_TODO] remove this condition once kubebuilder's CLI is
```
If we try to look for and search it in the code we will not found and it will not render in other colours in IDEs as well or allow its related features which can be helpful.

Reply:
Sure